src/brs-buildroot/patches/fwts/0007-add-brs-for-BRS-test-set.patch

```diff
@@ -1,252 +0,0 @@
-From fc2b19a664d7a7a3f65c26637ceaec3d9515e72b Mon Sep 17 00:00:00 2001
-From: Yin Wang <yin.wang@intel.com>
-Date: Thu, 14 Dec 2023 10:16:18 +0800
-Subject: [PATCH 7/7] add --brs for BRS test-set
-
----
- src/acpi/acpitables/acpitables.c |  2 +-
- src/acpi/madt/madt.c             | 11 ++++++++++-
- src/acpi/method/method.c         |  2 +-
- src/acpi/pptt/pptt.c             |  2 +-
- src/acpi/rsdp/rsdp.c             |  2 +-
- src/acpi/spcr/spcr.c             |  2 +-
- src/acpi/xsdt/xsdt.c             |  2 +-
- src/dmi/dmicheck/dmicheck.c      |  4 ++--
- src/lib/include/fwts_framework.h |  3 ++-
- src/lib/src/fwts_framework.c     | 13 ++++++++++++-
- src/sbbr/fadt/fadt.c             |  4 ++--
- src/sbbr/rsdp/rsdp.c             |  4 ++--
- 12 files changed, 36 insertions(+), 15 deletions(-)
-
-diff --git a/src/acpi/acpitables/acpitables.c b/src/acpi/acpitables/acpitables.c
-index bd99f5bb..55299cd8 100644
---- a/src/acpi/acpitables/acpitables.c
-+++ b/src/acpi/acpitables/acpitables.c
-@@ -365,6 +365,6 @@ static fwts_framework_ops acpi_table_check_ops = {
- };
- 
- FWTS_REGISTER("acpitables", &acpi_table_check_ops, FWTS_TEST_ANYTIME,
--	      FWTS_FLAG_BATCH | FWTS_FLAG_ACPI | FWTS_FLAG_SBBR)
-+	      FWTS_FLAG_BATCH | FWTS_FLAG_ACPI | FWTS_FLAG_SBBR | FWTS_FLAG_BRS)
- 
- #endif
-diff --git a/src/acpi/madt/madt.c b/src/acpi/madt/madt.c
-index 5eaf0694..de5f9695 100644
---- a/src/acpi/madt/madt.c
-+++ b/src/acpi/madt/madt.c
-@@ -126,6 +126,7 @@
- #define MAX_IO_APIC_ID		256 /* IO APIC ID field is 1 byte */
- 
- #define SBBR_ACPI_MAJOR_VERSION 6
-+#define BRS_ACPI_MAJOR_VERSION 6
- 
- struct acpi_madt_subtable_lengths {
- 	unsigned short major_version;	/* from revision in FADT header */
-@@ -411,6 +412,14 @@ static int madt_init(fwts_framework *fw)
- 			return FWTS_ERROR;
- 		}
- 	}
-+	if (fw->flags & FWTS_FLAG_BRS) {
-+		if (fadt_major < BRS_ACPI_MAJOR_VERSION) {
-+			fwts_log_error(fw, "BRS support starts with ACPI v6.0,"
```

Comment:
NIT: Actually ACPI support for RISC-V starts with 6.6 only. But here it looks like it checks only major version. But still message can be improved.